docs/deployment/teams.md

```diff
@@ -315,21 +315,47 @@ To set a member as a manager for a subteam:
 
 ## Appendices
 
-### Member-scoped access tokens
+### Tokens
 
-Both members and admins can log in through the command-line interface (CLI) by entering the following command:
+An access token is a secure credential that authorizes a user to interact with the Semgrep AppSec Platform or API without requiring a username and password. Access tokens identify who is making a request and define what actions they are allowed to perform, based on the [token’s scope](#token-scopes). 
```

Comment:
I'm slightly dubious about "identify who is making a request" here. My token can make a request without it being "me" making the request, which is one of the things we're trying to clarify in the text below. That said, I don't have a ready idea of what other phrase would make more sense.